CRM/Core/BAO/MessageTemplate.php

```diff
@@ -586,4 +586,25 @@ public static function sendTemplate($params) {
     return [$sent, $mailContent['subject'], $mailContent['text'], $mailContent['html']];
   }
 
+  /**
+   * Make a copy.
+   *
+   * @param int $id The entity id to copy.
+   * @param array $params
+   * @return CRM_Core_DAO
+   */
+  public static function copy($id, $params = []) {
+    $fieldsFix = [
+      'suffix' => [
+        'name' => ' - ' . ts('Copy'),
```

Comment:
@mattwire I don't think we have name column in message template table - we have field for `msg_title`.

Reply:
@yashodha I've fixed the `name` -> `msg_title`. I'm not sure that we need to append a number as well - other entities don't do that and we are not enforcing unique titles (only IDs).